app/services/doc_auth_router.rb

```diff
@@ -196,9 +196,14 @@ def self.client(vendor:, warn_notifier: nil)
   # rubocop:enable Layout/LineLength
 
   def self.doc_auth_vendor_for_bucket(bucket)
-    bucket == :alternate_vendor ?
-      IdentityConfig.store.doc_auth_vendor_randomize_alternate_vendor :
-      IdentityConfig.store.doc_auth_vendor
+    case bucket
+    when :socure
+      Idp::Constants::Vendors::SOCURE
+    when :lexis_nexis
```

Comment:
This is here since they are a vendor, even though they are currently set to the default.

Reply:
this would never get a hit in the current implementation right? we would have to add a :lexis_nexis bucket in DOC_AUTH_VENDOR, right?

Reply:
correct

Reply:
i'm wondering if there is a comment that could be added to make this clear ... so that someone could know that extra code is needed to make this bucket possible to be hit